src/style.css

```diff
@@ -1 +1,124 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  font-family: Avenir, sans-serif;
```

Comment:
You've specified a font-family here, but you haven't provided a fallback font. If the Avenir font fails to load for some reason, the browser will use the default font, which may not fit your design. It's a good idea to specify a generic font family that the browser can fall back to. For example: `font-family: Avenir, sans-serif;`.